src/state/nftMarket/types.ts

```diff
@@ -93,23 +87,43 @@ export interface NftTokenSg {
   collection?: {
     id: string
   }
+  updatedAt?: string
   transactionHistory?: Transaction[]
-  nftLocation?: NftLocation
 }
 
-export interface NFT {
-  tokenId: string
+// This type is somewhat specific to PancakeBunnies NFTs
+// It stores basic PancakeBunny NFT data together with `tokens` object containing specific token ID market data
+// It is used when retrieving all bunny tokens from redux.
+// In case you need to work with specific stand-alone token use NftToken
+export interface PancakeBunnyNftWithTokens {
```

Comment:
Also some fields like `collectionName` and `collectionAddress` were not populated at all. Again TS didn't complain cause of missing types..

Reply:
I am a bit confused on the name. This interface represents any NFT from any collection correct? I think to avoid confusion the name should be more generic.

Reply:
@hachiojidev Ideally in the future when we integrated Squads and other collections this should be reused, yes. But in current form its not clear if we'll able to just reuse it as is, some stuff is too PancakeBunny-specific, especially `tokens` field, right now its one of the main ones that we use in a lot of places but for Squads and other collection it won't exist at all.

as Hutch said in DM

> it makes it _super_ obvious what that type is (easy to find and replace all names later if we decide to)